.github/config.yml

```diff
@@ -0,0 +1,10 @@
+ # Comment to be posted to on PRs from first time contributors in your repository
+
+newPRWelcomeComment: >
+  Hello!👋 Thanks for contributing to the dbt product documentation and opening this pull request! ✨ 
+  
+We use Markdown and some HTML to write the dbt product documentation. When writing content, you can use our [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-types.md) to understand our writing standards and how we organize information in the dbt product docs.
```

Comment:
```suggestion
dbt Labs uses Markdown and some HTML to write the dbt product documentation. When writing content, you can use our [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [content types](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-types.md) to understand our writing standards and how we organize information in the product docs.
```

Reply:
I wonder if it sounds less robotic if we use "we?"

Reply:
"we" can be ambiguous. it can be "the author and the reader" or "the company" or something else entirely. sugg using "dbt Labs" for the first reference at least.

Reply:
I was intentionally breaking the rules to make the voice of the "bot" a little more inclusive of all the people who will contribute and have contributed to the docs, including the docs team, the community, and everyone at dbt. If it doesn't work, then definitely ✂️ cut it.